app/gateway/2.8.x/install-and-run/helm-quickstart-enterprise.md

```diff
@@ -0,0 +1,323 @@
+---
+title: How to Install Kong Gateway with Helm
+toc: false
+---
+
+This guide shows you how to deploy {{site.base_gateway}} with Helm on your local Kubernetes environment. These steps are known to work on [Docker Desktop Kubernetes](https://docs.docker.com/desktop/kubernetes/) and [Kind](https://kind.sigs.k8s.io/).
+Once deployed, Kong will be locally accessible at `https://kong.7f000001.nip.io`. [nip.io](https://nip.io) is used to automatically resolve this domain to localhost. 
+
+The {{site.base_gateway}} software is governed by the
+[Kong Software License Agreement](https://konghq.com/kongsoftwarelicense/).
+{{site.ce_product_name}} is licensed under an
+[Apache 2.0 license](https://github.com/Kong/kong/blob/master/LICENSE).
+
+
+{% navtabs %}
```

Comment:
Given the instructions are identical once you have an environment up and running, would it make sense to assume that they have a cluster running and `kubectl` configured correctly for this guide?

We can have another page explaining the options if they'd like a local cluster to test with.

Reply:
Could you clarify this point?

Reply:
The guides looked identical to me. On our call @usrbinkat mentioned that something around the ClusterIP/NodePort is different between the two so we couldn't combine